Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -15,70 +16,105 @@ class BRBViewController: UIViewController {
         case account(BRBAccount)
     }
     
-    private var connectionHandler: BRBConnectionHandler!
+    var accountManager: BRBAccountManager!
     private var requestStart: Date?
+    private var account: BRBAccount?
+    private var loggedIn = false
     
     private lazy var loginViewController = BRBLoginViewController()
-    private var accountViewController: BRBAccountViewController?
+    var accountViewController: BRBAccountViewController?
     
     private var state: State = .login
+    
+    private var activityIndicator: NVActivityIndicatorView?
+    var isLoading: Bool? {
```

Comment:
Can we get rid of `isLoading`? Seems all we use it for is to update the `activityIndicator`.

Reply:
We access isLoading in EateryTabBarController

Reply:
At the time I made this comment I hadn't seen `EateryTabBarController`, but with the additional comments I made below I still think we should remove this property.

Reply:
Wouldn't that mean we would have to unwrap activityIndicator a lot throughout the file? isLoading embeds that check, which is much cleaner I feel

Reply:
We can just do `activityIndicator?.startAnimating()`

Reply:
Wait, better suggestion. Make activity indicator not an optional. `let activityIndicator = NVActivityIndicatorView(< blah >)`